test/tests/test_logging/test_function_logs.sh

```diff
@@ -43,11 +42,18 @@ echo "Grabbing logs, should have 4 calls in logs"
 
 sleep 15
 
-logs=$(fission function logs --name $fn --detail)
+fission function logs --name $fn --detail > logfile
```

Comment:
Can you move this 'logfile' to /tmp? Also, can you add code to delete it before the test exits?

Reply:
sure thing